modules/alert_policy/main.tf

```diff
@@ -46,4 +46,38 @@ resource "opsgenie_alert_policy" "this" {
       }
     }
   }
+
+  dynamic time_restriction {
```

Comment:
Could you make use of this time_restriction in the examples so the tests can take advantage of this ?

https://github.com/cloudposse/terraform-opsgenie-incident-management/blob/master/examples/alert_policy/main.tf